src/ubx-message.toit

```diff
@@ -2800,13 +2797,18 @@ class CfgInf extends Message:
     return uint8_ 0
 
   /**
-  Enable or disable) a specific logging $level, for a specific $port.
+  Enable (or disable) a specific logging $level, for a specific $port.
+
+  Adds or removes the given $level bit from the ports' bitmask of enabled
+    logging levels.  Use $enable to set or unset the desired $level. (This
+    method does not replace other bits that might be set. In comparison,
+    $set-port-level replaces the entire bitmask with the new value).
 
   Sets the logging $level for all ports if $port is omitted. Logging $level
     should be one of  $LEVEL-ERROR, $LEVEL-WARNING, $LEVEL-NOTICE, $LEVEL-TEST,
     or $LEVEL-DEBUG.
   */
-  set-port-level port/int=PORT-ALL --level/int enable/bool=true -> none:
+  enable-port-level port/int=PORT-ALL level/int=LEVEL-ALL --enable/bool=true -> none:
```

Comment:
If the parameter is named, then keep the `set` name.
```suggestion
  set-port-level port/int=PORT-ALL level/int=LEVEL-ALL --enable/bool=true -> none:
```

feels nicer to have `set-port-level --enable` than `enable-port-level --enable`.